signUp.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LOG_IN</title>
+    <link rel="stylesheet" type="text/css" href="css/style1Signin.css" />
+    <script src="RESOURCES/CSS/JS/script.js"></script>
+  </head>
+  <body>
+    <div class="main-page">
+      <div class="navbar">
+        <div>
+          <img src="images/icon.png" alt="" />
+          <a href="index.html"><span>Algo</span> Phantom</a>
+        </div>
+        <nav>
+          <ul id="menuitems">
```

Comment:
Don't create a new navbar use the navbar present in `index.html` only.

Reply:
So in sign-up form there must be no nav-bar

Reply:
There can be a navbar but don't recreate just use the navbar that is present in the `index.html` so that the navbar is same throught the website.